app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [
+        my_favourite_films,
+        marks,
+        collection_of_coins
+    ],
+    "immutable": [
+        lucky_number,
+        pi,
+        one_is_a_prime_number,
+        name,
+        profile_info
+    ]
```

Comment:
the same here. do you really need every value in new line?

Reply:
flake8 writes that in this case lines are too long, more than 79 characters in one line